### PR TITLE
Py3

### PR DESCRIPTION
**Pull request by syphar**

### CODE REVIEW

**Comment by syphar** [comment]
@codingjoe 
- fixed the tests
- used a (IMHO) prettier version for the encoding problem (just use `response.text` from `requests`)
- bumped the version

File: closeio/utils.py [hunk 1]
```diff
@@ -25,25 +25,23 @@ def convert_errors():
...
+                error_message = e.response.text
 
-        if hasattr(e, 'response'):
             request = e.response.request
             request_data = 'url: {}\nbody: {}'.format(
                 request.url,
                 request.body,
             )
         else:
+            error_message = text_type(e)
```
Comment (codingjoe) [inline closeio/utils.py]: You can't just cast bytes to string. It will look like `u"b"asdf""`.

Comment (syphar) [inline closeio/utils.py]: that's an `SlumberBaseException`

**Comment by codingjoe** [comment]
Released in 0.1.43

**Pull request closed without merging**